src/main/resources/io/github/dgroup/arch4u/pmd/arch4u-template-ruleset.xml

```diff
@@ -77,4 +77,30 @@
       ]]>
     </example>
   </rule>
+
+  <rule name="DontCreateStringConstant"
+        since="0.4.0"
+        language="java"
+        message="Don''t create ''{0}'' String constant, use these classes instead: {1}"
```

Comment:
`message="Use existing constant instead of field, variable or direct string definition: https://github.com/dgroup/arch4u-pmd/discussions/43"`